dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -239,6 +242,21 @@ public List<Enrollment> getEnrollments(@Nonnull List<String> uids) throws Forbid
         .toList();
   }
 
+  @Override
+  public List<Enrollment> getEnrollments(
+      TrackedEntity trackedEntity, Program program, EnrollmentStatus enrollmentStatus)
+      throws ForbiddenException, BadRequestException {
+    EnrollmentOperationParams params =
```

Comment:
Because the org unit mode needs to be set, and depending on its value, an org unit needs to be provided too.
I don't want to leave it up to the caller to set these two values up, if I do it within this method I know the validation will pass.

Reply:
please mark it as deprecated for removal so we warn people 😅